### PR TITLE
Translation of us-US.po in french 

### PR DESCRIPTION
**Pull request by TuxSeb**
I translated the files "us-US.po" in french

### CODE REVIEW

File: jasper/data/locale/fr-FR.po [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: 2.0-dev\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-10-28 01:17+0100\n"
+"PO-Revision-Date: 2015-10-28 01:23+0100\n"
+"Last-Translator: Jan Holthuis <holthuis.jan@googlemail.com>\n"
+"Language-Team: Jasper Project\n"
+"Language: en_US\n"
```
Comment (Holzhaus) [inline jasper/data/locale/fr-FR.po]: Don't forget to edit Metadata (`Language`, `Last-Translator` and `PO-Revision-Date`).

**Comment by TuxSeb** [comment]
Ok, thanks !

**Comment by Holzhaus** [comment]
Thanks for contributing.

**Comment by TuxSeb** [comment]
Thank you ! I really like this project, it think i'll seriously contribute !

**Pull request merged**